sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_exporter.py

```diff
@@ -104,3 +162,158 @@ def shutdown(
         :param timeout_millis: The maximum amount of time to wait for shutdown. Not currently used.
         :type timeout_millis: float
         """
+
+
+    def _ping(self, monitoring_data_point) -> Optional[Response]:
+        ping_response = None
+        token = attach(set_value(_SUPPRESS_INSTRUMENTATION_KEY, True))
+        try:
+            ping_response = self._client.ping(  # type: ignore
+                monitoring_data_point=monitoring_data_point,
+                ikey=self._instrumentation_key,
+                x_ms_qps_transmission_time=_ticks_since_dot_net_epoch(),
+                cls=Response,
+            )
+            return ping_response  # type: ignore
+        except HttpResponseError:
+            # Errors are not reported
```

Comment:
nit: consider logging errors here. Same for Post.
In my testing, I was seeing errors like invalid ConnectionString. 

Surfacing these errors will be useful for your supportability.
Feel free to also leave a TODO here and address it in a follow up PR.